apps/arweave/src/ar_mining_worker.erl

```diff
@@ -199,24 +232,23 @@ handle_cast(handle_task, #state{ task_queue = Q } = State) ->
 			end
 	end;
 
-handle_cast({remove_sub_chunks_from_cache, SubChunkCount, Candidate}, State) ->
-	{noreply, remove_sub_chunks_from_cache(Candidate, SubChunkCount, State)};
+handle_cast(?MSG_REMOVE_SUB_CHUNKS_FROM_CACHE(SubchunkCount, Candidate), State) ->
+	{noreply, remove_subchunks_from_cache(Candidate, SubchunkCount, State)};
 
-handle_cast(check_worker_status, State) ->
+handle_cast(?MSG_CHECK_WORKER_STATUS, State) ->
 	maybe_warn_about_lag(State#state.task_queue, State#state.name),
-	maybe_warn_about_stale_chunks(State),
```

Comment:
`maybe_warn_about_stale_chunks` was just a stop gap when we didn't know what was causing the chunk cache to go negative, right? It's no longer needed?

Reply:
Not sure about that, I think I removed the warning we added earlier. If we don't need it I'll remove it, but I need to recalibrate the logic.